server/routes.go

```diff
@@ -963,7 +962,7 @@ func allowedHostsMiddleware(addr net.Addr) gin.HandlerFunc {
 		}
 
 		if allowedHost(host) {
-			if c.Request.Method == "OPTIONS" {
+			if c.Request.Method == http.MethodOptions {
```

Comment:
I'm fine turning off the linter however I disagree it's less clear. The var is much easier to search for and less error prone since typos causes build errors.